internal/migrations/sql/33_tutorials.up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TYPE TUTORIAL AS ENUM ('branding', 'agents', 'registry');
+
+CREATE TABLE UserAccount_Tutorial (
```

Comment:
The Go type for this table is called TutorialProgress. Would it make sense to use the same name here? Because I wasn't initially sure if they are the same type but I think they are (?)